From 579122369db49d45d555f66c26601431ccd127de Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sat, 22 Feb 2020 18:53:10 +0200
Subject: [PATCH 085/145] move network indicator

	modified:   packages/SystemUI/res/layout/status_bar.xml
	modified:   packages/SystemUI/res/layout/system_icons.xml
---
 packages/SystemUI/res/layout/status_bar.xml   | 16 ----------------
 packages/SystemUI/res/layout/system_icons.xml | 16 ++++++++++++++++
 2 files changed, 16 insertions(+), 16 deletions(-)

diff --git a/packages/SystemUI/res/layout/status_bar.xml b/packages/SystemUI/res/layout/status_bar.xml
index 22e82ae8df8..fe6ead447dc 100644
--- a/packages/SystemUI/res/layout/status_bar.xml
+++ b/packages/SystemUI/res/layout/status_bar.xml
@@ -109,22 +109,6 @@
             android:clipChildren="false"
             android:gravity="center_horizontal|center_vertical"/>
 
-        <com.android.systemui.statusbar.LineageStatusBarItemHolder
-            android:id="@+id/network_traffic_holder"
-            android:layout_width="wrap_content"
-            android:layout_height="match_parent"
-            android:gravity="start|center_vertical"
-            >
-
-            <org.lineageos.internal.statusbar.NetworkTraffic
-                android:id="@+id/network_traffic"
-                android:layout_width="wrap_content"
-                android:layout_height="match_parent"
-                android:maxLines="2"
-                android:gravity="center_vertical|start" />
-
-        </com.android.systemui.statusbar.LineageStatusBarItemHolder>
-
         <com.android.keyguard.AlphaOptimizedLinearLayout android:id="@+id/system_icon_area"
             android:layout_width="0dp"
             android:layout_height="match_parent"
diff --git a/packages/SystemUI/res/layout/system_icons.xml b/packages/SystemUI/res/layout/system_icons.xml
index 3e638599031..ed0a6f66597 100644
--- a/packages/SystemUI/res/layout/system_icons.xml
+++ b/packages/SystemUI/res/layout/system_icons.xml
@@ -21,6 +21,22 @@
     android:layout_height="match_parent"
     android:gravity="center_vertical">
 
+    <com.android.systemui.statusbar.LineageStatusBarItemHolder
+        android:id="@+id/network_traffic_holder"
+        android:layout_width="wrap_content"
+        android:layout_height="match_parent"
+        android:gravity="start|center_vertical"
+        >
+
+        <org.lineageos.internal.statusbar.NetworkTraffic
+            android:id="@+id/network_traffic"
+            android:layout_width="wrap_content"
+            android:layout_height="match_parent"
+            android:maxLines="2"
+            android:gravity="center_vertical|start" />
+
+    </com.android.systemui.statusbar.LineageStatusBarItemHolder>
+
     <com.android.systemui.statusbar.phone.StatusIconContainer android:id="@+id/statusIcons"
         android:layout_width="0dp"
         android:layout_weight="1"
-- 
2.17.1

